cucumber/authenticators_jwt/features/authn_jwt_ca_cert.feature

```diff
@@ -59,42 +60,38 @@ Feature: JWT Authenticator - ca-cert variable tests
     Examples:
-      | jwks-uri                                     |
-      | https://jwks/ca-cert.json                    |
-      | https://chained.mycompany.local/ca-cert.json |
+      | jwks-uri                                                |
+      | https://jwks/ca-cert-ONYX-15313.json                    |
+      | https://chained.mycompany.local/ca-cert-ONYX-15316.json |
 
   Scenario: ONYX-15314: Chained jwks-uri no ca-cert variable
-    Given I am the super-user
-    And I successfully set authn-jwt "jwks-uri" variable to value "https://chained.mycompany.local/ca-cert.json"
+    Given I initialize JWKS endpoint with file "ca-cert-ONYX-15314.json"
+    And I am the super-user
+    And I successfully set authn-jwt "jwks-uri" variable to value "https://chained.mycompany.local/ca-cert-ONYX-15314.json"
     When I GET "/authn-jwt/raw/cucumber/status"
     Then the HTTP response status code is 500
-    And the authenticator status check fails with error "CONJ00087E Failed to fetch JWKS from 'https://chained.mycompany.local/ca-cert.json'. Reason: '#<OpenSSL::SSL::SSLError: SSL_connect returned=1 errno=0 state=error: certificate verify failed (self signed certificate in certificate chain)>'>"
+    And the authenticator status check fails with error "CONJ00087E Failed to fetch JWKS from 'https://chained.mycompany.local/ca-cert-ONYX-15314.json'. Reason: '#<OpenSSL::SSL::SSLError: SSL_connect returned=1 errno=0 state=error: certificate verify failed (self signed certificate in certificate chain)>'>"
 
-  @skip
   @sanity
   Scenario: ONYX-15315: Self-signed jwks-uri with valid ca-cert variable value
-    Given I am the super-user
+    Given I initialize JWKS endpoint with file "ca-cert-ONYX-15315.json"
+    And I am the super-user
     And I extend the policy with:
     """
     - !variable conjur/authn-jwt/raw/ca-cert
     """
-    And I successfully set authn-jwt "jwks-uri" variable to value "https://chained.mycompany.local/ca-cert.json"
+    And I successfully set authn-jwt "jwks-uri" variable to value "https://chained.mycompany.local/ca-cert-ONYX-15315.json"
     And I fetch root certificate from https://chained.mycompany.local endpoint as "chained"
     And I successfully set authn-jwt "ca-cert" variable value to the "chained" certificate
     When I GET "/authn-jwt/raw/cucumber/status"
     Then the HTTP response status code is 200
     And the HTTP response content type is "application/json"
     And the authenticator status check succeeds
 
-  Scenario: ONYX-15317: Google's jwks-uri no ca-cert variable
-    Given I am the super-user
-    And I successfully set authn-jwt "jwks-uri" variable to value "https://www.googleapis.com/oauth2/v3/certs"
-    When I GET "/authn-jwt/raw/cucumber/status"
-    Then the HTTP response status code is 200
-    And the HTTP response content type is "application/json"
-    And the authenticator status check succeeds
-
-  @skip
+  # ONYX-15318 and ONYX-15317 are order sensitive tests
```

Comment:
Seems like a bug not?

Reply:
Not. It's a proper work of cache.
If conjur already fetched keys from some url, keys remains in memory. So even there's no connectivity to jwks endpoint or like in example, I've added ca-cert variable with wrong cert chain; keys are already in memory - conjur does not invoke a request to bring them...